configurations/sharepoint-woo/synchronizations/sharepoint-woo-verzoeken-to-publications.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Sharepoint Woo-verzoeken en -besluiten to Publications",
+	"description": "",
+	"version": "0.0.2",
```

Comment:
shouldn't this be 0.0.1?
What about covenant? Or maybe other publication types?